flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestUpdateDeleteTableFactory.java

```diff
@@ -123,6 +131,20 @@
                     .withDescription(
                             "The columns' name for the required columns in row-level delete");
 
+    private static final ConfigOption<List<String>> REQUIRED_COLUMNS_FOR_UPDATE =
+            ConfigOptions.key("required-columns-for-update")
+                    .stringType()
+                    .asList()
+                    .noDefaultValue()
+                    .withDescription("The name for the required columns in row-level update");
+
+    private static final ConfigOption<Boolean> ONLY_REQUIRE_UPDATED_COLUMNS_FOR_UPDATE =
+            ConfigOptions.key("only_require_updated_columns_for_update")
```

Comment:
'only_require_updated_columns_for_update' -> 'only-require-updated-columns-for-update'